ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-next__4.10.yaml

```diff
@@ -0,0 +1,315 @@
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/pizzaplanetv1
+  to: knative-serving-test-pizzaplanetv1
+- dockerfile_path: openshift/ci-operator/knative-test-images/wsserver/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/wsserver
+  to: knative-serving-test-wsserver
+- dockerfile_path: openshift/ci-operator/knative-test-images/httpproxy/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/httpproxy
+  to: knative-serving-test-httpproxy
+- dockerfile_path: openshift/ci-operator/knative-test-images/singlethreaded/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/singlethreaded
+  to: knative-serving-test-singlethreaded
+- dockerfile_path: openshift/ci-operator/knative-test-images/servingcontainer/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/servingcontainer
+  to: knative-serving-test-servingcontainer
+- dockerfile_path: openshift/ci-operator/knative-test-images/sidecarcontainer/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/sidecarcontainer
+  to: knative-serving-test-sidecarcontainer
+- dockerfile_path: openshift/ci-operator/knative-test-images/hellohttp2/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/hellohttp2
+  to: knative-serving-test-hellohttp2
+- dockerfile_path: openshift/ci-operator/knative-test-images/hellovolume/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/hellovolume
+  to: knative-serving-test-hellovolume
+- dockerfile_path: openshift/ci-operator/knative-test-images/readiness/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/readiness
+  to: knative-serving-test-readiness
+- dockerfile_path: openshift/ci-operator/knative-test-images/emptydir/Dockerfile
+  from: base
+  inputs:
+    test-bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/emptydir
+  to: knative-serving-test-emptydir
+promotion:
+  name: knative-nightly
+  namespace: openshift
+releases:
+  initial:
+    integration:
+      name: "4.10"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.10"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+test_binary_build_commands: make test-install
+tests:
+- as: e2e-aws-ocp-410
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.8"
```

Comment:
this is wrong I believe. It has to be 4.10. If this is the same in 4.9, we have a bug.

Reply:
Ugh... Probably we need to wait for 4.10 is available in https://github.com/openshift/release/tree/master/clusters/hive/pools/openshift-ci/ ?

Reply:
Or alternatively run without pooling for 4.10 while there's no pool yet.

Reply:
Okay, let's ping to test-platform team on Slack if they can easily add the pool for 4.10 or not. If not, let's run without pooling for 4.10.

Reply:
As per test-platform team, 4.10 is not available until stable 4.10 is available in https://amd64.ocp.releases.ci.openshift.org/#4-stable

@Kaustubh-pande Could you try to run without pooling for 4.10? I think you can refer to other repo's setting or this PR https://github.com/openshift/release/pull/22681